data/helpers.d/psql

```diff
@@ -0,0 +1,150 @@
+# Create a master password and set up global settings
+# Please always call this script in install and restore scripts
+#
+# usage: ynh_psql_test_if_first_run
+ynh_psql_test_if_first_run() {
+	if [ -f /etc/yunohost/psql ];
+	then
+		echo "PostgreSQL is already installed, no need to create master password"
+	else
+		pgsql=$(ynh_string_random)
+		pg_hba=""
+		echo "$pgsql" >> /etc/yunohost/psql
+
+		if [ -e /etc/postgresql/9.4/ ]
+		then
+			pg_hba=/etc/postgresql/9.4/main/pg_hba.conf
+		elif [ -e /etc/postgresql/9.6/ ]
+		then
+			pg_hba=/etc/postgresql/9.6/main/pg_hba.conf
+		else
+			ynh_die "postgresql shoud be 9.4 or 9.6"
```

Comment:
Why ?
What's going to happen with the future versions ?

Reply:
We could also get the version by something like : 
`dpkg -l | grep -P -o "postgresql\s*\d\.\d" | grep -P -o "\d\.\d"`

Reply:
I guess we'll need to update the helpers because we can't predict how things are going to be yet.

Reply:
Still, why 9.4 or 9.6 only?

Reply:
9.4 was the Jessie version, 9.6 is the Stretch version. Still, a comment must be added, and the helpers will need to be upgraded at next Debian migration (analogous to php-fpm helpers).